### PR TITLE
Retcon some gravatar/Racker metadata onto my posts

### PR DESCRIPTION
**Pull request by lvh**
@smashwilson kindly showed me how to make my Gravatar work and get that nice Racker ribbon; this PR adds that to my older posts.

### CODE REVIEW

File: _posts/2016-05-12-supersingular-isogeny-diffie-hellman-101.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ title: "Supersingular isogeny Diffie-Hellman 101"
 date: 2016-05-12 12:15
 comments: false
 author: lvh
+authorIsRacker: true
```
Comment (smashwilson) [inline _posts/2016-05-12-supersingular-isogeny-diffie-hellman-101.md]: This one already had `authorIsRacker` set! Looks like the build is complaining about the duplicate.

File: _posts/2015-08-31-using-querySelector-on-elements.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ title: Using querySelector and querySelectorAll on Elements
 date: '2015-08-31 23:59'
 comments: true
 author: lvh
+authorIsRacker: true
+authorAvatar: https://s.gravatar.com/avatar/1846c8040fcf70e9b55bb7bfcdb78bc4
```
Comment (smashwilson) [inline _posts/2015-08-31-using-querySelector-on-elements.md]: Also, I think you have to quote these so the `:` doesn't confuse the YAML parser.

**Comment by lvh** [comment]
Thanks, @smashwilson! I've addressed your comments, hopefully the builder likes this better.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-d5ceb3de81/). :bow:

**Comment by smashwilson** [comment]
:metal:

**Pull request merged**